### PR TITLE
NO-ISSUE: MicroShift Footprint and performance: don't send result on presubmit, use correct branch

### PR DESCRIPTION
**Pull request by pmtk**

### CODE REVIEW

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-footprint-and-performance

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-release-4.17-footprint-and-performance

**Comment by pmtk** [comment]
/hold
work in progress

**Comment by pmtk** [comment]
/pj-rehearse pull-ci-openshift-microshift-main-footprint-and-performance pull-ci-openshift-microshift-release-4.17-footprint-and-performance periodic-ci-openshift-microshift-release-4.17-footprint-and-performance-nightly periodic-ci-openshift-microshift-release-4.18-periodics-footprint-and-performance-nightly

**Comment by pmtk** [comment]
/pj-rehearse periodic-ci-openshift-microshift-release-4.18-periodics-footprint-and-performance-nightly pull-ci-openshift-microshift-main-footprint-and-performance periodic-ci-openshift-microshift-release-4.17-footprint-and-performance-nightly

File: ci-operator/step-registry/openshift/microshift/e2e/footprint-and-performance/openshift-microshift-e2e-footprint-and-performance-commands.sh [hunk 1]
```diff
@@ -90,4 +90,10 @@ while true ; do
     sleep 30
 done
 
-ssh "${INSTANCE_PREFIX}" 'bash -x $HOME/microshift//scripts/ci-footprint-and-performance/3-test.sh'
+BRANCH=$(echo $CLONEREFS_OPTIONS | jq -r '.refs[] | select(.repo=="microshift") | .base_ref')
```
Comment (ggiguash) [inline ci-operator/step-registry/openshift/microshift/e2e/footprint-and-performance/openshift-microshift-e2e-footprint-and-performance-commands.sh:93]: Can you add an example / comment on what CLONEREFS_OPTIONS looks like?

File: ci-operator/step-registry/openshift/microshift/e2e/footprint-and-performance/openshift-microshift-e2e-footprint-and-performance-commands.sh [hunk 2]
```diff
@@ -90,4 +90,40 @@ while true ; do
...
+#             "base_ref": "main",
+#             "workdir": true
+#         }
+#     ],
+#     "fail": true
+# }
+BRANCH=$(echo $CLONEREFS_OPTIONS | jq -r '.refs[] | select(.repo=="microshift") | .base_ref')
+if [[ "${BRANCH}" == "" ]]; then
+    env
+    exit 1
```
Comment (ggiguash) [inline ci-operator/step-registry/openshift/microshift/e2e/footprint-and-performance/openshift-microshift-e2e-footprint-and-performance-commands.sh:126]: Should we add an error message here?

**Pull request merged**